.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      files: 'python/*'
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
This is a good start, but you want to make sure everything is being checked, including yaml files. yaml is a separate hook id